docs/4.0/trustedclusters.md

```diff
@@ -192,8 +209,8 @@ spec:
     - remote: "admin"
       local: [local-admin]
   token: "join-token"
-  tunnel_addr: main.example.com:3024
-  web_proxy_addr: main.example.com:3080
+  tunnel_addr: <main-proxy-address>:3024
```

Comment:
I'd probably put this back to `main.example.com` to be consistent with the rest of the docs.